docs/toml-schema.md

```diff
@@ -430,3 +430,16 @@ allowed-merge-teams = ["awesome-team"]
 # (optional)
 merge-bots = ["homu"]
 ```
+
+### Crates.io trusted publishing
+Configure crates.io Trusted Publishing for crates published from a given repository from GitHub Actions.
+
+```toml
+[[trusted-publishing]]
+# Name of the crate that will be published from this repository (required)
+crate = "regex"
+# Name of a GitHub Actions workflow file that will publish the crate (required)
+workflow-filename = "ci.yml"
+# GitHub Actions environment that has to be used for the publishing (optional)
+environment = "deploy"
```

Comment:
Only people with maintain+ access on a repo can create environments. We'd need a way to let people create environments without infra-admins involvement. Also, migrating all environments to be managed by the team repo might not be trivial, as a lot of environments nowadays are created by Terraform.